Ginger/Ginger/ALM/ALMDefectsProfilesPage.xaml.cs

```diff
@@ -241,15 +216,35 @@ private void RefreshgrdDefectsProfilesHandler(object sender, RoutedEventArgs e)
 
         private void RefreshgrdDefectsFieldsHandler(object sender, RoutedEventArgs e)
         {
-            Mouse.OverrideCursor = Cursors.Wait;
             ALMDefectProfile AlmDefectProfile = (ALMDefectProfile)grdDefectsProfiles.CurrentItem;
-            ALMIntegration.Instance.UpdateALMType(AlmDefectProfile.AlmType);
-            AlmDefectProfile.ALMDefectProfileFields = ALMIntegration.Instance.GetALMItemFieldsREST(true, ALM_Common.DataContracts.ResourceType.DEFECT, null);
+            {
+                mALMDefectProfileFields = FetchDefectFields(AlmDefectProfile.AlmType);
+                mALMDefectProfileFields.Where(z => z.Mandatory != true).ToList().ForEach(x => x.SelectedValue = string.Empty);
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5717441)